client/log_verifier.go

```diff
@@ -48,7 +48,7 @@ func NewLogVerifier(hasher hashers.LogHasher, pubKey crypto.PublicKey, sigHash c
 		Hasher:  hasher,
 		PubKey:  pubKey,
 		SigHash: sigHash,
-		v:       merkle.NewLogVerifier(hasher),
+		v:       logverifier.NewLogVerifier(hasher),
```

Comment:
`logverifier.New` would look more Go-style. Note that you don't need to preserve the original name because you have an alias.

Reply:
done